index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+
```

Comment:
Kindly add the font link here or the beginning of the CSS as I requested on my first review. Please check [this](https://www.w3schools.com/css/css_font_google.asp) or [this link](https://linuxhint.com/google-fonts-html/#:~:text=To%20add%20google%20fonts%2C%20search,section%20of%20the%20HTML%20file.) to learn how to add a font to your project.
Right now your fonts look like this below
<img width="374" alt="Screenshot 2022-12-02 at 2 25 36 PM" src="https://user-images.githubusercontent.com/25887253/205283371-a87b6bd6-622d-41e6-a15d-c8d28ecd33aa.png">

But Figma design look like this below. Check how the fonts look different.

<img width="462" alt="Screenshot 2022-12-02 at 2 31 10 PM" src="https://user-images.githubusercontent.com/25887253/205283473-e6fceaf8-29ed-4f37-9e2e-25b2feeecee7.png">